files/lib/system/conversation/ConversationHandler.class.php

```diff
@@ -122,4 +123,33 @@ public function getConversationCount($userID = null) {
 		
 		return $this->conversationCount[$userID];
 	}
+	
+	/**
+	 * Enforces the flood control.
+	 */
+	public function enforceFloodControl() {
+		$limit = WCF::getSession()->getPermission('user.conversation.maxStartedConversationsPer24Hours');
+		if ($limit < 1) {
```

Comment:
```suggestion
		if ($limit < 0) {
```
Otherwise `0` would also be interpreted as unlimited.